lib/new-admin/config/accounts.js

```diff
@@ -50,7 +51,8 @@ const ALL_ACCOUNTS = [
   { code: 'mailgun', display: 'Mailgun', class: EMAIL },
   { code: 'none', display: 'None', class: ZERO_CONF, cryptos: [BTC, ZEC, LTC, DASH, BCH, ETH, XMR] },
   { code: 'blockcypher', display: 'Blockcypher', class: ZERO_CONF, cryptos: [BTC] },
-  { code: 'mock-zero-conf', display: 'Mock 0-conf', class: ZERO_CONF, cryptos: ALL_CRYPTOS, dev: true }
+  { code: 'mock-zero-conf', display: 'Mock 0-conf', class: ZERO_CONF, cryptos: ALL_CRYPTOS, dev: true },
+  { code: 'ciphertrace', display: 'CipherTrace', class: WALLET_SCORING, cryptos: [BTC, ETH, LTC, BCH] }
```

Comment:
Just a reminder for what we discussed.

Reply:
After our call, this is to stay. Resolving this thread